agatemulator.txt

```diff
@@ -1,19 +1,19 @@
 [Section]
 Name = Agat Emulator
-Version = 1.27
+Version = 1.29.1
 License = GPL
 Description = Apple ][ and Agat 7/9 software simulation program for Win32.
-Size = 6.0 MiB
+Size = 6.1 MiB
 Category = 15
 URLSite = http://agatemulator.sourceforge.net/
-URLDownload = http://download.sourceforge.net/project/agatemulator/agatemulator/1.27/agatemulator-1.27.exe
-SHA1 = 4584110ca12dcb1536cc675624716b4c0e0ace6d
+URLDownload = https://sourceforge.net/projects/agatemulator/files/agatemulator/1.29.1/agatemulator-1.29.1.exe
+SHA1 = 99db84ac514ea588d8d4938f09a9d0a0749ab461
 CDPath = none
-SizeBytes = 6314919
+SizeBytes = 6346282
```

Comment:
This is a new field, the idea is that it will be added everywhere the `Size` is now.
For backwards compatibility, we will still maintain the old `Size` field.

Reply:
I see, I have updated all `Size` entries as well.